photon-client/src/main/scala/com/linkedin/photon/ml/data/avro/AvroUtils.scala

```diff
@@ -90,9 +228,9 @@ object AvroUtils {
    * @param record The input Avro record
    * @return The nameAndTerm parsed from the Avro record
    */
-  protected[ml] def readNameAndTermFromGenericRecord(record: GenericRecord): NameAndTerm = {
-    val name = Utils.getStringAvro(record, AvroFieldNames.NAME)
-    val term = Utils.getStringAvro(record, AvroFieldNames.TERM, isNullOK = true)
+  def readNameAndTermFromGenericRecord(record: GenericRecord): NameAndTerm = {
```

Comment:
I can put `protected` back - in the midst of the refactoring, it was messing with tests, then I forgot about it.